calour/sorting.py

```diff
@@ -195,10 +196,38 @@ def sort_by_metadata(exp, field, axis=0, inplace=False):
         x = exp.feature_metadata
     else:
         raise ValueError('unknown axis %s' % axis)
-    idx = np.argsort(x[field].values, kind='mergesort')
+    idx = _argsort(x[field].values)
     return exp.reorder(idx, axis=axis, inplace=inplace)
 
 
+def _argsort(values):
```

Comment:
we should move this to util.py. make it accept an iterable and yield the tuple as in the solution sent to you.